test/parallel/test-http-parser-consume.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+const spawn = require('child_process').spawn;
+
+if (process.argv[2] === 'child') {
+  var server = http.createServer(function(q, s) {
+    s.end('hello');
+  });
+
+  server.listen(common.PORT, function(s) {
+    var rr = http.get('http://localhost:' + common.PORT, function(d) {
```

Comment:
Could the first parameter just be `{ port: server.address().port }`. Also, please use `const`.